src/core/src/service_interfaces/LifecycleManagerAzure.py

```diff
@@ -96,6 +96,8 @@ def execution_start_check(self):
 
             # Signalling take-over of core state by auto-assessment after safety checks for any competing process
             self.update_core_sequence(completed=False)
+            # Refresh status file in memory to be up-to-date
+            self.status_handler.load_status_file_components()
```

Comment:
1. Should this be done on line 120 so it is done for both cases?
2. Should it be done in all code paths within update_core_sequence? Update_core_sequence has an early return.
3. What about all invocations of update_core_sequence? Should it be done there as well?

Reply:
1. The second case should already have up-to-date status file information because there isn't any delay there. It isn't needed on line 120 but can be moved there.
2. Core sequence is read at the beginning of each loop within the `execution_start_check` method, so it should always have up-to-date information.
3. I don't think this is needed. It's only needed in areas of contention when there are multiple core processes running at the same time, and this is already taken care of (as mentioned above).